web_infrastructure/jenkins_plugin.py

```diff
@@ -408,11 +408,18 @@ def _get_installed_plugins(self):
 
     def install(self):
         changed = False
+
         plugin_file = (
             '%s/plugins/%s.jpi' % (
                 self.params['jenkins_home'],
                 self.params['name']))
 
+        hpi_file = '%s/plugins/%s.hpi' % (
+            self.params['jenkins_home'],
+            self.params['name'])
+        if os.path.isfile(hpi_file):
+            os.rename(hpi_file, hpi_file+'.bak')
```

Comment:
@jtyr See the note I added with Jenkins issue. I left it in so there is some recourse if they want to revert.